README.md

```diff
@@ -44,6 +44,12 @@ Defaults to `*:*:*`.
 * `foo:bar:baz`
 * `a:b:c x:y:z`
 
+##### Optional: opts
```

Comment:
Line 30 should be updated to include this

```diff
- #### `jwt([String scope])` -> `Promise<String>`
+ #### `jwt([String scope][, Object opts])` -> `Promise<String>`
```

Reply:
Also, we didn't think through the API very well so write now it behaves a bit weird. We have two optional parameters, but count on `opts` being the second one.

We should add a check in local.js to allow you to only specify `opts` (vs `jwt(undefined, { extendSession: false })`)

```js
if (typeof scope === 'object') {
  opts = scope;
  scope = undefined;
}
```

Reply:
Ahh yes, I see what you are saying, I will make both changes specified here. There is a `DEFAULT_SCOPE` defined in the file, so I will set scope to that vs. `undefined` as specified in the code block above:

```
scope = scope || DEFAULT_SCOPE;

if (typeof scope === 'object') {
	opts = scope;
	scope = DEFAULT_SCOPE;
}